webapp/messages/en.json

```diff
@@ -1,4 +1,7 @@
 {
+  "metadata": {
+    "title": "Hemi Portal"
+  },
```

Comment:
the locale files files are sorted by key. Can we place this in the appropriate line? It should go ~151/152, just before `navbar` (there is an issue somewhere to add something that sort this automatically 😄 )

Reply:
Ah yep! Thanks

Reply:
Done, thank you!